src/components/user/tour/tour.js

```diff
@@ -1,6 +1,67 @@
 import React from "react";
-import Typography from "@material-ui/core/Typography";
+import { DataGrid } from '@material-ui/data-grid';
 
-export default function Tour() {
-  return <Typography paragraph>The 'Tour' Page comming soon...</Typography>;
+const rows = [
+  {
+    id: 1, 
+    status: 'All booked',
+    name: 'Bellinis in Armenia',
+    origin: 'Italy',
+    days: '7',
+    date: 'Jan 28-Feb 11',
+    destination: 'Yerevan, Sevan, Dilijan',
+    guests: 'Marco Belini, Chiara Belini',
+    hotels: 'Noy Land, Popock 1',
+    activities: 'Yell Park, Papanino',
+    flight: 'QR 706',
+    price: '$5400',
+    notes: 'Very picky person',
+    createdBy: 'Armine G',
+   },
+   {
+    id: 1, 
+    status: 'All booked',
+    name: 'Bellinis in Armenia',
+    origin: 'Italy',
+    days: '7',
+    date: 'Jan 28-Feb 11',
+    destination: 'Yerevan, Sevan, Dilijan',
+    guests: 'Marco Belini, Chiara Belini',
+    hotels: 'Noy Land, Popock 1',
+    activities: 'Yell Park, Papanino',
+    flight: 'QR 706',
+    price: '$5400',
+    notes: 'Very picky person',
+    createdBy: 'Armine G',
+   },
+];
+
+const columns = [
+  { field: 'status', headerName: 'Status'},
+  { field: 'name', headerName: 'Name', width: 150 },
+  { field: 'origin', headerName: 'Origin' },
+  { field: 'days', headerName: 'Nr. of days' },
+  { field: 'destination', headerName: 'Destination', width: 150 },
+  { field: 'date', headerName: 'Dates', width: 150 },
+  { field: 'guests', headerName: 'Guests', width: 150 },
+  { field: 'hotels', headerName: 'Hotels', width: 150 },
+  { field: 'activities', headerName: 'Activities', width: 150 },
+  { field: 'flight', headerName: 'Flight' },
+  { field: 'price', headerName: 'Price' },
+  { field: 'notes', headerName: 'Notes' },
+  { field: 'createdBy', headerName: 'Created By' },
+];
+
+export default class Tour extends React.Component {
+  constructor(props) {
+    super(props);
+    };
+
+  render() {
+    return (
+      <div style={{ height: 600, width: '100%', top: '40px' }}>
+        <DataGrid rows={rows} columns={columns} />
```

Comment:
[autoHeight](https://material-ui.com/api/data-grid/#props )

It will be more convenient to use `autoHeight` property. The grid will render as many rows as you have in `rows={rows}` prop.

There is no need to pass a `style` prop for parent `div`. The grid will calculate height automatically and will grow parent height as well. If you have only one component as a child, you can skip the wrapper. 
BTW: When you pass a style prop and pass an object literal, you force to rerender component each time, because each time you create a new Object with a new reference.